client/landing/gutenboarding/stores/onboard/types.ts

```diff
@@ -41,6 +41,7 @@ export interface Design {
 	src: string;
 	template: string;
 	theme: string;
+	preview?: string;
```

Comment:
@lsl not a blocker, just thinking about whether the `preview` type could be annotated like this instead:

```typescript
preview?: 'static';
```

Just so when we want to add another one of these we don't have to dig around in the code to remember what the special value of the flag is supposed to be.